File: 2232.c
#include <math.h> 
#include <stdio.h> 

int main() { 
    int i, j, t, n, res; 
    
    scanf("%d", &t); 

    for(i = 0; i < t; i++) { 
        scanf("%d", &n); 
        res = 0; 
        for(j = 0; j < n; j++) { 
            res += pow(2, j); 
        } 
        
        printf("%d\n", res); 
    } 
    
    return 0; 
}

File: 2232.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int t = sc.nextInt();
        for (int i = 0; i < t; i++) {
            int n = sc.nextInt();
            int res = 0;
            for (int j = 0; j < n; j++) {
                res += Math.pow(2, j);
            }
            System.out.println(res);
        }

        sc.close();
    }
}
